unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Editor/compiledVersion.gen.txt

```diff
@@ -0,0 +1 @@
+1.0.0-20220525130536.commit-7f8e347
```

Comment:
This should not be ignored or in `Generated` folder?

Reply:
No, this is the compiled version, we need to push this to the repo. This will be the the file that the system will look to compile new protos